src/gtfs/repository/ScheduleBuilder.ts

```diff
@@ -88,7 +97,7 @@ export class ScheduleBuilder {
       routeTypeIndex.hasOwnProperty(row.train_category) ? routeTypeIndex[row.train_category] : RouteType.Rail,
```

Comment:
I'll ask Alistair whether any of other modes than Rail can have 1st class
```
  Tram = 0,
  Subway = 1,
  Rail = 2,
  Bus = 3,
  Ferry = 4,
  Cable  = 5,
  Gondola = 6,
  Funicular = 7
```
Because I'm not sure he mentioned that only buses cannot have it.